auto_nag/scripts/survey_sec_bugs.py

```diff
@@ -75,4 +75,5 @@ def comment_tpl_for_bugid(self, bugid):
 
 
 if __name__ == "__main__":
-    SurveySecurityBugs().run()
+    return
+    # SurveySecurityBugs().run()
```

Comment:
I think deleting [run_survey_sec_bugs.sh](https://github.com/mozilla/relman-auto-nag/blob/master/run_survey_sec_bugs.sh) is more suitable to avoid unnecessary execution (e.g., update deps and check logs). 

For this, I guess something needs to be done for the scheduled cron jobs. 

@marco-c wdyth?

Reply:
We could just comment everything in run_survey_sec_bugs.sh